message.go

```diff
@@ -102,12 +102,16 @@ func requestDataForMessage(originator string, recipients []string, body string,
 	} else {
 		request.MClass = 1
 	}
+
+	if params.ScheduledDatetime.IsZero() == false {
```

Comment:
To make it consistent with our existing code base, would you mind simplifying if statement:
`if !params.ScheduledDatetime.IsZero() {`